pkg/metrics/metrics.jsx

```diff
@@ -484,6 +484,7 @@ class CurrentMetrics extends React.Component {
                                         value={info.use} min={0} max={100}
                                         className="pf-m-sm"
                                         variant={info.use > 90 ? ProgressVariant.danger : ProgressVariant.info}
+                                        isTitleTruncated
```

Comment:
Ugh, that is a rather weird/bad attribute name, but indeed that's what the [official docs](https://www.patternfly.org/v4/components/progress/) say. Do you have a screenshot? I'd actually rather not truncate it at all and let it line-wrap, but *if* it gets truncated, then (1) at the left (as usually the rightmost directory is the more recognizable/important part), and (2) it needs to have a tooltip with the complete name.

Also, "trancate" typo in the commit message.

Reply:
Good point here!
PF4 adds a tooltip to the truncated string automatically. 
But PF4 does not offer truncation from left. This can be easily done by adding 'direction: rtl;' in our CSS rules though. I don't have a strong opinion, but to be honest I find it just fine how it was. I am leaning towards just closing the issue without this fix with the truncation.

Reply:
I'm fine with just closing it, and seeing if someone complains.